packages/glibc.rb

```diff
@@ -85,30 +117,34 @@ def self.build
       system build_env.transform_keys(&:to_s), '../configure', *config_opts, no_preload_hacks: true
       system "make PARALLELMFLAGS='-j #{CREW_NPROC}'", no_preload_hacks: true
     end
-
-    arch_flag = case ARCH
-                when 'x86_64'
-                  '-m64'
-                when 'i686'
-                  '-m32'
-                else
-                  '-mfloat-abi=hard -mfpu=vfpv3-d16 -march=armv7-a+fp'
-                end
-
-    # Link with ChromeOS's glibc libraries to ensure compatibility
-    system <<~CMD, chdir: 'crew-package-glibc/crew-preload', no_preload_hacks: true
-      cc -B #{CREW_GLIBC_PREFIX} #{CREW_COMMON_FLAGS} #{arch_flag} -fuse-ld=mold \
-        #{cc_macro_list.join(' ')} ../prebuilt/#{ARCH.sub('aarch64', 'armv7l')}/lib{c,dl}-*.so \
-        -shared -fvisibility=hidden -Wl,-soname,crew-preload.so \
-        main.c hooks.c -o ../crew-preload.so
-    CMD
   end
 
   def self.install
-    FileUtils.mkdir_p %W[#{CREW_DEST_PREFIX}/etc/env.d #{CREW_DEST_PREFIX}/etc/ld.so.conf.d #{CREW_DEST_LIB_PREFIX}]
+    FileUtils.mkdir_p %W[#{CREW_DEST_PREFIX}/etc/env.d #{CREW_DEST_PREFIX}/etc/ld.so.conf.d #{CREW_DEST_PREFIX}/share #{CREW_DEST_LIB_PREFIX}]
 
     system "make DESTDIR=#{CREW_DEST_DIR} install", chdir: 'builddir'
-    system "make DESTDIR=#{CREW_DEST_DIR} localedata/install-locales", chdir: 'builddir'
+
+    @interpreter = case ARCH
+                   when 'aarch64', 'armv7l'
+                     "#{CREW_DEST_PREFIX}/opt/glibc-libs/ld-linux-armhf.so.3"
```

Comment:
Add `"#{CREW_DEST_PREFIX}/opt/glibc-libs/ld-linux-aarch64.so.3"``?

Reply:
This isn't building a aarch64 glibc at the moment, so that doesn't make sense. We don't even have a native aarch64 toolchain in our repo.